rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-11-09
+nightly
```

Comment:
probably should lock to a specific nightly. not sure which to target, but it needs to be newer than what we were at to get these targets

Reply:
I'll check if there's any particular nightly we should be pinning to; otherwise I'll just pick "last week" or something.

Reply:
I pinned it to today's arbitrarily, but can tweak if there's a specific one